subprojects/docs/src/samples/userguide/dependencies/iteratingArtifacts/build.gradle

```diff
@@ -0,0 +1,22 @@
+repositories {
+    jcenter()
+}
+
+configurations {
+    scm
+}
+
+dependencies {
+    scm 'org.eclipse.jgit:org.eclipse.jgit:4.9.2.201712150930-r'
+    scm 'commons-codec:commons-codec:1.7'
+}
+
+//START SNIPPET iteration-task
+task iterateScmArtifacts {
+    doLast {
```

Comment:
These sort of tasks need to have dependencies on the configuration or else composite builds may not work.

Reply:
Thanks for the info. I think that's important information to have on the page on composite builds. I will change the code as such:

```
task iterateScmArtifacts {
     dependsOn configurations.scm
     ...
}
```

Reply:
done